src/main/java/com/facebook/openwifirrm/aggregators/MeanAggregator.java

```diff
@@ -8,28 +8,46 @@
 
 package com.facebook.openwifirrm.aggregators;
 
+import java.util.LinkedList;
+import java.util.List;
+
 /**
  * Tracks the mean of all added values. If no values are added, the mean is 0.
  */
 public class MeanAggregator implements Aggregator<Double> {
```

Comment:
General question, do we only care about average for the whole duration? What about mean of a time frame?

Reply:
I guess now we only care about average for the whole duration. RSSI, MCS are all discrete type. `MeanAggregator` operates in a given time frame(That is, time frame is not a parameter in the calculation). I guess only when the variables are continuous we need to calculate mean of a time frame, but all the continuous type are decomposed in records.